crypto/bls/indy_crypto/bls_crypto_indy_crypto.py

```diff
@@ -60,25 +42,56 @@ def _prepare_seed(seed):
 
         return seed_bytes
 
+
+class IndyCryptoMultiSigVerifier:
+
+    def __init__(self, generator):
```

Comment:
I think it's better to pass GroupParams instead of generator to the constructor (generator is implementation details, let's not be so specific)